api/src/test/java/io/opencensus/stats/ViewTest.java

```diff
@@ -70,7 +70,7 @@ public void testIntervalView() {
   public void testViewEquals() {
     new EqualsTester()
         .addEqualityGroup(
-            View.create(NAME, DESCRIPTION, MEASURE, MEAN, KEYS, Cumulative.create()),
+            View.create(NAME, DESCRIPTION, MEASURE, MEAN, KEYS),
             View.create(NAME, DESCRIPTION, MEASURE, MEAN, KEYS, Cumulative.create()))
```

Comment:
Should this `Cumulative.create()` be removed too?

Reply:
I left it in order to assert that the new and old factory methods give the same result for cumulative views.